### DISCUSSION TITLE
Does not respond when using bash command <(curl -s https://pterodactyl-installer.se)

### DISCUSSION THREAD

**Discussion by AdamRoyn** [vilhelmprytz/pterodactyl-installer#341] [Q&A] [answered]
I'm using vps Ubuntu 20.04
![Screenshot_20220627-230555_Admin Hands](https://user-images.githubusercontent.com/58103343/175989432-236e57e2-276a-48e4-9545-903d015a4900.png)

**Comment by vilhelmprytz** [comment, marked as answer]
There are no known outages with https://pterodactyl-installer.se currently. Can you try just `curl https://pterodactyl-installer.se` and see wht it says?

**Reply by AdamRoyn** [reply]
```curl: (6) Could not resolve host: pterodactyl-installer.se```

**Reply by Linux123123** [reply]
You don't have internet connection / your dns server doesn't work.

**Reply by AdamRoyn** [reply]
How to fix it?

**Comment by AdamRoyn** [comment]
![IMG-20220628-WA0011](https://user-images.githubusercontent.com/58103343/176089702-426fad9d-7b68-4ddc-9adb-f30d0793feef.jpg)
it works but this error appears

**Reply by vilhelmprytz** [reply]
Can you provide more information/logs? I'd recommend the [Discord server](https://pterodactyl-installer.se/discord) for support with the script.
